semcore/data-table/src/Body.tsx

```diff
@@ -58,12 +59,59 @@ class Body extends Component<AsProps, State> {
   firstRowRef = React.createRef<HTMLDivElement>();
   firstRowResizeObserver: ResizeObserver | null = null;
 
+  lockedCell: [HTMLElement | null, boolean] = [null, false];
+
   getRowHeight = () => {
     const { virtualScroll } = this.asProps;
     const rowHeightFromProps = typeof virtualScroll === 'object' && virtualScroll?.rowHeight;
     return rowHeightFromProps || this.state.rowHeight;
   };
 
+  handleKeyDown = (e: React.KeyboardEvent) => {
+    if (e.currentTarget === this.lockedCell[0]) {
+      const focusableChildren = Array.from(this.lockedCell[0].children).flatMap((node) =>
```

Comment:
Another general comment: I found that in Columns merging example you can't jump from unmerged non-first cell to a merged cell (on screenshot you can't go up) 
<img width="704" alt="Screenshot 2024-09-12 at 18 34 53" src="https://github.com/user-attachments/assets/70cdf52d-f114-45e0-991f-f8cc0ef68721">

Reply:
yes, this will be fixed in the next versions